docs/modules/kubernetes/partials/nav.adoc

```diff
@@ -1,7 +1,8 @@
 * Kubernetes
 ** xref:kubernetes:deploying-in-kubernetes.adoc[Overview]
-** xref:operator:ROOT:get-started.adoc[Hazelcast Platform Operator]
+** xref:kubernetes:deploying-in-kubernetes.adoc#hazelcast-platform-operator-for-kubernetesopenshift[Hazelcast Platform Operator]
 ** xref:kubernetes:kubernetes-auto-discovery.adoc[Kubernetes Auto Discovery]
+** xref:kubernetes:kubernetes-persistence.adoc[Kubernetes Persistence]
```

Comment:
Title of section is `Kubernetes`, I think we can use `Persistence` here directly. WDYT?


<img width="230" alt="Screenshot 2024-03-07 at 16 15 30" src="https://github.com/hazelcast/hz-docs/assets/6005622/8f62d00e-ab4f-4393-bac2-5ff852eb963a">

Reply:
I think the section probably needs a more thorough restructure but these two label fixes are worth making